features/woocommerce-payments.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * WooCommerce Payments
+ *
+ * @package jurassic-ninja
+ */
+
+namespace jn;
+
+const WCPAY_DEFAULTS = array(
+	'woocommerce-payments-dev-tools' => false,
+	'woocommerce-payments-jn-options' => false,
+	'woocommerce-payments-release' => false,
+);
+
+add_action(
+	'jurassic_ninja_init',
+	function () {
+		add_action(
+			'jurassic_ninja_add_features_before_auto_login',
+			function ( &$app = null, $features, $domain ) {
+				$features = array_merge( WCPAY_DEFAULTS, $features );
+				if ( ! isset( $features['woocommerce-payments'] ) && $features['woocommerce-payments-release'] ) {
+					debug( '%s: Adding WooCommerce Payments (version %s)', $domain, $features['woocommerce-payments-release'] );
+					add_woocommerce_payments_release_plugin( $features['woocommerce-payments-release'] );
+				}
+
+				if ( $features['woocommerce-payments-dev-tools'] ) {
+					debug( '%s: Adding WooCommerce Payments Dev Tools', $domain );
+					add_woocommerce_payments_dev_tools();
+				}
+
+				if ( $features['woocommerce-payments-jn-options'] ) {
+					debug( '%s: Adding WooCommerce Payments Jurassic Ninja Options', $domain );
+					add_woocommerce_payments_jurassic_ninja_options();
+				}
+			},
+			10,
+			3
+		);
+
+		add_filter(
+			'jurassic_ninja_rest_feature_defaults',
+			function ( $defaults ) {
+				return array_merge( $defaults, WCPAY_DEFAULTS );
+			}
+		);
+
+		add_filter(
+			'jurassic_ninja_rest_create_request_features',
+			function ( $features, $json_params ) {
+				if ( isset( $json_params['woocommerce-payments-dev-tools'] ) ) {
+					$features['woocommerce-payments-dev-tools'] = true;
+				}
+
+				if ( isset( $json_params['woocommerce-payments-jn-options'] ) ) {
+					$features['woocommerce-payments-jn-options'] = true;
+				}
+
+				if ( isset( $json_params['woocommerce-payments-release'] ) ) {
+					$features['woocommerce-payments-release'] = $json_params['woocommerce-payments-release'];
```

Comment:
```suggestion
					$features['woocommerce-payments-release'] = $json_params['woocommerce-payments-release'];
					$features['woocommerce'] = true;
```
It would be handy to auto-install WC if WC Payments is requested.